python/pyspark/sql/readwriter.py

```diff
@@ -147,6 +147,7 @@ def load(self, path=None, format=None, schema=None, **options):
                        or a DDL-formatted string (For example ``col0 INT, col1 DOUBLE``).
         :param options: all other string options
 
+        >>> spark.conf.set("spark.sql.sources.default", "parquet")
```

Comment:
Can we just call `format('parquet')` like the doctest for JSON below?

Reply:
Yep. That was my first commit [here](https://github.com/apache/spark/pull/20705#discussion-diff-171729865R150). I'll rollback this.